clients/admin-ui/src/features/plus/plus.slice.ts

```diff
@@ -61,9 +67,19 @@ export const plusApi = createApi({
     }),
     getAllClassifyInstances: build.query<
       ClassifyInstanceResponseValues[],
-      void
+      ClassifyInstancesParams
     >({
-      query: () => `classify/`,
+      query: (params) => {
+        const urlParams = new URLSearchParams();
+        urlParams.append("resource_type", params.resource_type);
+        params.fides_keys?.forEach((key) => {
+          urlParams.append("fides_keys", key);
+        });
```

Comment:
I couldn't find an easier way to do this but I'm all ears...

Reply:
This is the canonical way to have an "array" in search/query params, and yes it strange. Looks like RTK supports a passing a simple object out of the box, but not a _fun_ case like this. https://redux-toolkit.js.org/rtk-query/api/fetchBaseQuery#paramsserializer